types.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'tripcode' {
+  export default function tripcode(key: string): string;
```

Comment:
Double checking: is `default` correct here if we're using `module.exports =`? I thought we might need something like:
```typescript
declare function tripcode(key: string): string;
export = tripcode;
```

Reply:
Yes, you are right. I have re-read the documentation on .d.ts, and it says to use the old `export =` syntax for better compatibility. Although in modern versions of the TS both syntaxes seem to work the same